api/pkg/cli/utils/utils.go

```diff
@@ -0,0 +1,108 @@
+// Copyright © 2021 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"fmt"
```

Comment:
instead of `utils` package I guess we can move this to `options` package. Also we can rename `select_options` package to `options` and move this over there. WDYT?

Reply:
Hub cli commands already have a struct called options I think it will overlap with that 🤔

Reply:
`options` package I am saying